maths.py

```diff
@@ -4,7 +4,7 @@ def fact(num):
     return num * fact(num - 1)
 
 def sq(num):
-    return num*num
+    return num ** 2
```

Comment:
Why are you returning square?
```suggestion
    return num * num * num
```

Reply:
Because this is the sq function. What did you expect?